### PR TITLE
update unittest for authentidication

### PR DESCRIPTION
**Pull request by Zaplatynskyi**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777&view=detail&selectedIssue=LEAR-84)

## Code reviewers

- [ ] @Sokolovskyi1995
- [ ] @BDubas
- [ ] @anastasiyaSht
- [x] @rromanitc
- [x] @martasakal
- [ ] @aberezh

### Second Level Review

- [ ] @lhalam
- [ ] @ssOleg

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by martasakal — DISMISSED** [review]

**Review by anastasiyaSht — DISMISSED** [review]

**Review by rromanitc — DISMISSED** [review]

File: eventually/tests/unittests/authentication/testmodel.py [hunk 1]
```diff
@@ -75,20 +70,30 @@ def test_get_by_id_positive(self):
         self.assertEqual(user_returned.updated_at, TEST_DATE)
         self.assertEqual(user_returned.is_active, False)
 
-
+    def test_get_by_id_with_cache_positive(self):
+        """Positive test of the CustomUser.get_by_id() method"""
+        with mock.patch('authentication.models.cache') as mock_cache:
+            with mock.patch('authentication.models.cache') as mock_cache:
```
Comment (aberezh) [inline eventually/tests/unittests/authentication/testmodel.py]: 75 і 76 рядки повторюються

File: eventually/tests/unittests/authentication/testmodel.py [hunk 2]
```diff
@@ -163,28 +187,44 @@ def test_to_dict(self):
...
-        expected_value = CustomUser.objects.all()
-        current_value = CustomUser.get_all()
-        self.assertEqual(list(current_value), list(expected_value))
+        with mock.patch('authentication.models.cache') as mock_cache:
+            with mock.patch('pickle.loads') as mock_pickle:
+                mock_cache.__contains__.return_value = True
+                mock_pickle.return_value = CustomUser.objects.all()
+                expected_value = CustomUser.objects.all()
+                current_value = CustomUser.get_all()
+                self.assertEqual(list(current_value), list(expected_value))
```
Comment (aberezh) [inline eventually/tests/unittests/authentication/testmodel.py]: Може краще використати "assertListEqual"?

File: eventually/tests/unittests/authentication/testmodel.py [hunk 3]
```diff
@@ -163,28 +187,44 @@ def test_to_dict(self):
...
+
+    def test_get_all_users_pickle(self):
+        """ Test of the CustomUser.get_all() method """
+        with mock.patch('pickle.loads') as mock_pickle:
+            mock_pickle.return_value = CustomUser.objects.all()
+            expected_value = CustomUser.objects.all()
+            current_value = CustomUser.get_all()
+            self.assertEqual(list(current_value), list(expected_value))
```
Comment (aberezh) [inline eventually/tests/unittests/authentication/testmodel.py]: І тут assertListEqual?

**Review by aberezh — DISMISSED** [review]

**Review by lhalam — CHANGES_REQUESTED** [review]
Update to comments!

**Review by anastasiyaSht — DISMISSED** [review]

**Review by anastasiyaSht — DISMISSED** [review]

**Review by martasakal — DISMISSED** [review]

**Review by BDubas — DISMISSED** [review]

**Review by lhalam — DISMISSED** [review]

**Review by lhalam — APPROVED** [review]

**Review by BDubas — APPROVED** [review]

**Review by rromanitc — APPROVED** [review]

**Review by anastasiyaSht — APPROVED** [review]

**Review by martasakal — APPROVED** [review]

**Pull request merged**